src/Radio.tsx

```diff
@@ -41,6 +42,7 @@ const StyledRadio = styled.input`
   cursor: pointer;
 
   ${props => props.disabled && `cursor: not-allowed;`}
+  ${getGlobalFocusStyles(0)};
```

Comment:
Is it possible to make the focus ring a circle for radios? 
<img width="105" alt="CleanShot 2022-05-04 at 07 27 38@2x" src="https://user-images.githubusercontent.com/4608155/166703092-1fd9a4c3-335a-4cdf-a698-5e73c4f16270.png">

Reply:
There is, but I'm matching what we have in Primer CSS.